concepticondata/conceptlists/Luangthongkum-2019-341.tsv

```diff
@@ -0,0 +1,342 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Luangthongkum-2019-341-1	1	all, all gone, whole		
+Luangthongkum-2019-341-2	2	ant	587	ANT
+Luangthongkum-2019-341-3	3	arrive	1387	ARRIVE
+Luangthongkum-2019-341-4	4	arrow, dart	977	ARROW
+Luangthongkum-2019-341-5	5	ascend, go up, rise	3207	GO UP OR RISE (ONESELF)
```

Comment:
I think this is rather  RISE (MOVE UPWARDS) as GO DOWN (DESCEND) is in the list as well.

Reply:
See also: https://github.com/concepticon/concepticon-data/pull/876#discussion_r448861281

Reply:
RISE (MOVE UPWARDS) refers to heavenly bodies like the sun, the moon, etc. This is rather specific.

Reply:
Yes, but the current mapping specifically refers not to heavenly bodies while I would be pretty sure that this entry refers to them.